PowerShellTools/Commands/UserInterface/ParameterEditorViewModel.cs

```diff
@@ -25,6 +25,7 @@ internal sealed class ParameterEditorViewModel : ObservableObject, IDisposable
         public ParameterEditorViewModel(ParameterEditorModel model)
         {
             _model = Arguments.ValidateNotNull(model, "model");
+            _model.CommonParameters = _model.CommonParameters.OrderBy(param => param.Name).ToList();

```

Comment:
Yes, the size changes correctly between versions. (assuming this is what you want...). 
![50](https://cloud.githubusercontent.com/assets/12631548/8069977/f7fc429e-0eb2-11e5-96d6-614f10a624ec.PNG)
![40](https://cloud.githubusercontent.com/assets/12631548/8069979/f998e2a6-0eb2-11e5-920d-41eea813564c.PNG)
Will fix the .Net style guideline for String vs string. As for the sorting, alright, in the xaml it shall go! I did look into doing in xaml earlier though and I wasn't getting anywhere with the examples I could find. Thought doing it at ViewModel was at the same "layer" so went with that instead. I'll give it a second shot.